dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -299,6 +299,16 @@ META_DATA_EXCEPTION.useraction=Verify that the metadata is enabled on the backen
 # 7019 deleted
 # 7020 reserved for other NLS file
 # 7021-7042 deleted
+
+# Do not translate: java.util.Logging
+ORACLE_TRACE_PARSE_WARNING=DSRA7043W: A default value was used for the system property {0} because of parsing error: {1}
+ORACLE_TRACE_PARSE_WARNING.explanation=The system property strings for Oracle are parsed and passed to the java.util.Logging library.  If a property cannot be parsed a default value is used.
+ORACLE_TRACE_PARSE_WARNING.useraction=Examine the exception, and fix the DataSource configuration.
```

Comment:
```suggestion
ORACLE_TRACE_PARSE_WARNING.useraction=Examine the exception and fix the DataSource configuration.
```

is there any more specific advice we can give- for example, are oracle properties specified in a particular element(s) in the DataSource config?

Reply:
Should be "and fix the trace configuration" because these aren't DataSource properties